core/sdk-core/src/main/java/software/amazon/awssdk/core/client/builder/SdkDefaultClientBuilder.java

```diff
@@ -372,6 +373,7 @@ public final B overrideConfiguration(ClientOverrideConfiguration overrideConfig)
         overrideConfig.advancedOption(ENDPOINT_OVERRIDDEN_OVERRIDE).ifPresent(value -> {
             clientConfiguration.option(ENDPOINT_OVERRIDDEN, value);
         });
+        clientConfiguration.option(CLIENT_OVERRIDE_CONFIGURATION, overrideConfig);
```

Comment:
This duplicates data that was already pulled out elsewhere. To prevent duplication, can we just pull out what we need here (similar to ENDPOINT_OVERRIDDEN.... SIGNER_OVERRIDDEN?)

Reply:
Sure; I added this change before I merged from `master` where `ENDPOINT_OVERRIDDEN_OVERRIDE` was! I'll copy the same pattern